app/models/passenger.rb

```diff
@@ -9,6 +9,10 @@ class Passenger < ApplicationRecord
   scope :active, -> { where(active: true) }
   scope :inactive, -> { where(active: false) }
 
+  # TODO: Make configurable by user
+  MOBILITY_DEVICES = ['Boot', 'Crutches', 'Cane', 'Walker',
```

Comment:
I was asking @Anbranin this question earlier... doesn't a wheelchair count as a mobility device?